### PR TITLE
update appearance of the calendar

### PR DESCRIPTION
**Pull request by sapirshp**

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -9,4 +9,5 @@
     <string name="Morning">Morning\n</string>
     <string name="Afternoon">Afternoon\n</string>
     <string name="Evening">Evening\n</string>
+    <string name="when_can_you_meet">When can you meet?</string>
```
Comment (Chen-Ben-yashar) [inline app/src/main/res/values/strings.xml]: I think it is a good idea to add the days here as strings... 
they do not change.
is it possible, or is it problematic?

File: app/src/main/java/com/example/meetapp/GroupActivity.java [hunk 2]
```diff
@@ -118,7 +125,7 @@ public int compare(Map.Entry<TimeSlot, Integer> o1,
     }
 
     public void displayTopSelections(){
-        TextView selectionsDisplayTextView = findViewById(R.id.selectionDisplay);
+//        TextView selectionsDisplayTextView = findViewById(R.id.selectionDisplay);
```
[the remark this answers is not in the document]
Comment (sapirshp) [reply app/src/main/java/com/example/meetapp/GroupActivity.java]: fixed

File: app/src/main/java/com/example/meetapp/GroupActivity.java [hunk 3]
```diff
@@ -129,14 +136,18 @@ public void displayTopSelections(){
             topSelections = topSelections + topSlot.getDate() + " " + topSlot.getHour() + " - " +
                     + slotSelections.get(topSlot) + "/" + membersAmount + "\n";
         }
-        selectionsDisplayTextView.setText(topSelections);
+//        selectionsDisplayTextView.setText(topSelections);
```
[the remark this answers is not in the document]
Comment (sapirshp) [reply app/src/main/java/com/example/meetapp/GroupActivity.java]: fixed

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (sapirshp) [reply app/src/main/res/values/strings.xml]: right now the days are in a map that the key is the day number (i.e "22") and the value is the day (i.e "Sat")

Comment (sapirshp) [reply app/src/main/res/values/strings.xml]: fixed

**Pull request merged**